src/lib/viewers/image/MultiImage.scss

```diff
@@ -33,7 +33,10 @@
 .bp-images {
     border: 0 none;
     margin: 0 auto;
-    padding: 0;
+    // Since we use a box-shadow to denote focus, we need to add padding as images appear on top of box-shadow
+    padding: 7px;
+    // Allows focus to apply to the scrollable content on click
+    pointer-events: none;
```

Comment:
To ask the obvious question: Do we have any cases where images are expecting click events to work? Image maps or something? 😄